### ISSUE TITLE
Can't be installed on version 6.12.8

### ISSUE THREAD

**Issue by sepshow** [Leseratte10/unraid-plugin-bind-all#2]
Can't be installed on version 6.12.8

**Comment by Leseratte10** [comment]
Yeah, that's because I hardcoded the plugin to <= 6.12.6. It manipulates quite a bit of network code and I didn't want people to install it on an untested version (and also was kinda hoping Limetech would fix that issue quickly ...)

I'll try to get this tested on 6.12.8 today or tomorrow and update the plugin.

**Comment by Leseratte10** [comment]
Gave it a quick test on a 6.12.8 machine, seems to be working, so I increased the max version to 6.12.8. Let me know if there's any issues still.

**Comment by sepshow** [comment]
Good job！thx
Install it later for testing

**Comment by sepshow** [comment]
> Gave it a quick test on a 6.12.8 machine, seems to be working, so I increased the max version to 6.12.8. Let me know if there's any issues still.

Repeated the test several times, and found a problem after the plugin was successfully installed, whenever unraid performs a reboot or shutdown. The plugin will be listed in the "Plugin File Install Errors" option.
As a result, the plugin will not work, and you will need to delete it in the "Plugin File Install Errors" option and reinstall it.

Screenshot of successful installation:
![网页捕获_5-3-2024_16959_172 16 5 28](https://github.com/Leseratte10/unraid-plugin-bind-all/assets/52785574/ce0f9bc1-e535-4577-a240-67fe058489f8)

Take a screenshot after reboot or shutdown：
![网页捕获_5-3-2024_161830_172 16 5 28](https://github.com/Leseratte10/unraid-plugin-bind-all/assets/52785574/d455e216-1dfd-42b0-89e0-55ec1bc3a323)

**Comment by Leseratte10** [comment]
Interesting - I cannot reproduce the issue. Installed the plugin on a new 6.12.8 installation and rebooted the machine multiple times, it always loads correctly and I can access the WebUI over IPv6 even after a reboot. 

Can you remove the plugin, reboot the machine and install it again? Maybe the update failed somehow. If that still doesn't work, can you post your diagnostics zip file?

**Comment by sepshow** [comment]
Of course you can, I'll test it again later.

**Comment by sepshow** [comment]
Interesting - Now it's normal. 
Thanks bro!
